sp800-63b/sec4_aal.md

```diff
@@ -82,6 +82,8 @@ Communication between the claimant and verifier (the primary channel in the case
 
 Verifiers operated by government agencies at AAL2 SHALL be validated to meet the requirements of [FIPS 140](#FIPS140-2) Level 1.
 
+When a device such a smartphone is used in the authentication process, the unlocking of that device (typically done using a PIN or biometric) SHALL NOT be considered to satisfy one of the authentication factors. This is because it is generally not possible for verifier to know that the device had been locked, nor whether the unlock process met the requirements for the relevant authenticator type.
```

Comment:
Well pressing home button and/or swiping is still unlocking in my simple mind.  Maybe any efforts to access the desktop/home...It's a nit, but this is fun :)